src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -70,6 +73,11 @@
     private static final String STACK_TRACE_ON_USER_EXCEPTION_PROPERTY = "GATK_STACKTRACE_ON_USER_EXCEPTION";
 
     /**
+     * The option specifying a main configuration file.
+     */
+    private static final String gatkConfigFileOption = "--gatk-config-file";
```

Comment:
Maybe this should be in `StandardArgumentDefinitions` for avoid repetition in other parts of the code (adding a comment about where it is used).